README.md

```diff
@@ -10,6 +10,83 @@ git clone https://github.com/cambiatus/contracts.git
 
 Our docker setup include two nodes `keosd` and `nodeosd`. The `keosd` is responsible for running the wallet and as well the `cleos` commmand, since this way `cleos` won't spin up its own version of `keosd` daemon everytime you invoke it. The other `nodeos` node is responsible for running the blockchain and compiling files.
 
+## Build
+
+Building the Community Contract
+
+```
+cd community
+make
+```
+
 ## Attention
 
 This repo only works using [eosio.cdt](https://github.com/EOSIO/eosio.cdt/releases/tag/v1.7.0) version `v1.7.0`
+
+## Setting up a Local EOS Environment
+
+To setup a local environment you need to install locally a version of EOSIO;
+if installed properly, it will add `nodeos` and `cleos` to your binaries.
+
+Then you can execute the following commands:
+
+```sh
+WALLET_PASSWORD="PW5JZy495U6ATmFSFUmh7zBj7RDbW9fERKtwmcPbAWc75efVeKQ35"
```

Comment:
This is dependent of the setup, so I think its safe to remove this password, or at least add instructions on how to create the wallet and save the password to a file. Allowing us to do stuff like:

```sh
cleos wallet unlock --password $(cat file-with-password)
```

Reply:
good catch! will leave it to the user